starter-code/css/main.css

```diff
@@ -6,3 +6,189 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  /* height: 100vh; */
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.navbar {
+  display: flex;
+  height: 60px;
+  width: 100%;
+  padding: 10px;
+  justify-content: space-between;
+  align-items: center;
+  position: fixed;
+  font-size: small;
+  background-color: white;
+  z-index: 99;
+  color: 1a1a1a;
+}
+
+.links {
+  display: flex;
+  align-items: flex-end;
+}
+
+li {
+  list-style-type: none;
+  display: flex;
+  padding: 0 5px;
+}
+
+li a {
+  text-decoration: none;
+  color: black;
+}
+
+ul {
+  display: flex;
+}
+
+.main-section {
+  margin-top: 70px;
+}
+
+.spotify-logo {
+  height: 100%;
+}
+
+h1 {
```

Comment:
nice job guys!
Small note, avoid using h1 or h2 in css and use classes, identifiers and other type of selectors instead.

Reply:
Thanks for feedback :)
No h1 and h2 in css ever again 👍